.circleci/config.yml

```diff
@@ -459,6 +459,19 @@ jobs:
       FORCE_RELEASE: ON
       MAKEFLAGS: -j 10
 
+  b_static: &build_static
```

Comment:
Why not `b_ubu_static`?

Reply:
Was not clear that it will be ubuntu after all - will change!